Client/Frontend/Sync/SyncAddDeviceViewController.swift

```diff
@@ -203,31 +201,12 @@ class SyncAddDeviceViewController: SyncViewController {
         enterWordsButton.setTitleColor(BraveUX.GreyH, for: .normal)
         enterWordsButton.addTarget(self, action: #selector(SEL_showCodewords), for: .touchUpInside)
 
-        let buttonsStackView = UIStackView(arrangedSubviews: [UIView.spacer(.horizontal, amount: 16),
-                                                              doneEnterWordsStackView,
-                                                              UIView.spacer(.horizontal, amount: 16)])
-        buttonsStackView.setContentCompressionResistancePriority(UILayoutPriority(rawValue: 1000), for: .vertical)
+        doneEnterWordsStackView.setContentCompressionResistancePriority(UILayoutPriority(rawValue: 1000), for: .vertical)
```

Comment:
Use `.required` instead of `UILayoutPriority(rawValue: 1000)`

Reply:
thanks, this view has a lot of outdated code like this :)